dbms/src/Functions/divide.cpp

```diff
@@ -62,7 +62,11 @@ struct TiDBDivideFloatingImpl<A, B, false>
     template <typename Result = ResultType>
     static Result apply(A a, B b)
     {
-        return static_cast<Result>(a) / b;
+        // round for decimal floating number dividing
```

Comment:
Can you provide a more detailed comment?
for example
```
ref https://github.com/pingcap/tiflash/issues/6462, 
int / decimal ........
.....
```

Reply:
added